styles/capsule.json

```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
+	"version": 3,
+	"title": "Capsule",
+	"slug": "capsule",
```

Comment:
We need to come up with a consistent system for the slugs and for organizing the style files in subfolders.
https://github.com/WordPress/twentytwentyfive/issues/3

I have already added a very similar style
https://github.com/WordPress/twentytwentyfive/blob/trunk/styles/blocks/post-terms-1.json

I wonder if we can use the same style for both blocks? I am not sure if the inconstancies in the border color is intentional, like it is meant to be different for the visual hierarchy? @beafialho

Reply:
Ah, I just remembered that the terms one need to style the **links** only, since there are multiple individual items. So while it will have very similar styling it needs to be separate style variations, unless we want to add **custom CSS** that can be reused for both.

Reply:
Thanks for looking at this @carolinan - and for the resources on fullsiteediting.com as until I started this I wasn't even aware we could add style variations as json until I read it on there! 🙂 

It's a shame we can't reuse the same style without custom CSS. I assume we'd prefer to stick with style variations to push this as the recommended convention to use?

I think the subfolder organisation makes sense, but the slug naming could be tricky to apply to block styles. For example, if we opt for `heading-1` here, we're ignoring the fact that this style _could_ be reused, on paragraphs for example, by extending the `blockTypes` array.

I understand the portability aspect relating to section styles, but perhaps block style variations _should_ be named according to the type of styling they apply, as they're so specific. Clearly yourself and others have already been discussing/thinking about this at length, so I'm happy to make whatever changes you suggest here.

Reply:
I think this should not block the rest of the PR. Since these styles, for now, are not used in many templates or patterns, changing the slug later is not a blocker.